spec/bourbon/css3/font_face_spec.rb

```diff
@@ -32,14 +34,16 @@
                 "format(\"embedded-opentype\"), url(\"/assets.woff2\") " +
                 "format(\"woff2\"), url(\"/assets.woff\") format(\"woff\"), " +
                 "url(\"/assets.ttf\") format(\"truetype\"), " +
-                "url(\"/assets.svg#Georgia\") format(\"svg\");; " +
+                "url(\"/assets.svg#Georgia\") format(\"svg\"); " +
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.